src/limit-add.ts

```diff
@@ -0,0 +1,10 @@
+export const limitAdd = (firstValue: number, secondValue: number, limit: number) => {
+    const result = firstValue + secondValue;
+
+    if (result <= limit) {
+        return result;
+    }
+    else {
```

Comment:
You don't need the `else` here because `return` stops the code from executing any further

```
if (thing == 1) {
    return;
}

console.log("bla bla!");
```

if `thing` is `1`, it will go into the `if` block and return, so it won't get to the `console.log` line.

You can instead just do:

```
if (result <= limit) {
     return result;
}

return limit;
```